games/s.yaml

```
@@ -1960,6 +1960,25 @@
   - >-
     https://raw.githubusercontent.com/swatteau/sokoban-rs/master/assets/image/screenshot.png
 
+- name: SolarConflict
+  originals:
+  - Spacewar!
+  type: clone
+  repo: 'https://github.com/jtpalmer/sdl-contest/tree/master/week1'
```

```suggestion
  repo: https://github.com/jtpalmer/sdl-contest
```
osgc doesn't handle monorepos well